api4/status.go

```diff
@@ -137,6 +137,7 @@ func updateUserCustomStatus(c *Context, w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	customStatus.SetDefaults()
 	customStatus.TrimMessage()
```

Comment:
1. Combine these 2 functions.
2. Maybe rename this to `customStatus.PreSave()` similar to `post` and other models.